tests/data/general_filtering/basic_codelist.yaml

```diff
@@ -0,0 +1,10 @@
+- Big Variable:
+    definition: Some basic variable
+    required: True
+- Another Variable:
+    definition: some details
+    required_A: True

```

Comment:
Please use other attribute names, `required_A` and `required_B` doesn't really fit, maybe `some_attribute`...